contrib/networkmanager-openconnect/template.py

```diff
@@ -0,0 +1,44 @@
+pkgname = "networkmanager-openconnect"
+pkgver = "1.2.10"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = ["--with-gtk4=yes"]
+make_cmd = "gmake"
+make_dir = "."
+hostmakedepends = [
+    "automake",
+    "glib-devel",
+    "libtool",
+    "pkgconf",
+    "file",
+    "intltool",
+    "gettext-devel",
+    "gcr-progs",
+    "gmake",
+]
+makedepends = [
+    "libxml2-devel",
+    "libsecret-devel",
+    "gtk+3-devel",
+    "libnma-devel",
+    "gcr-devel",
+    "webkitgtk-devel",
+    "openconnect-devel",
+    "networkmanager-devel",
+    "gtk4-devel",
+]
+pkgdesc = "OpenConnect support for NetworkManager"
+maintainer = "Erica Z <zerica@callcc.eu>"
+license = "GPL-2.0-or-later"
+url = "https://gitlab.gnome.org/GNOME/NetworkManager-openconnect"
+source = f"{url}/-/archive/{pkgver}/NetworkManager-openconnect-{pkgver}.tar.bz2"
+sha256 = "df21a8730438b1614de390ecf1f73d379536d388a8e464c9a802dab14dd23c8f"
+options = ["!lintstatic"]
```

Comment:
this is wrong, it probably makes a /usr/lib/networkmanagert plugin with .a because of defaultlibs, put --default-library=shared in configure_args so it doesn't

Reply:
o rite yea, misread as meson initially